cmd/rekor-cli/app/verify.go

```diff
@@ -164,8 +164,11 @@ var verifyCmd = &cobra.Command{
 			}
 		}
 
-		// Note: the returned entry UUID is the UUID (not include the Tree ID)
-		leafHash, _ := hex.DecodeString(o.EntryUUID)
+		outputUUID, err := sharding.GetUUIDFromIDString(o.EntryUUID)
```

Comment:
Would verification in cosign also need to be updated to take into account that it might get an EntryID instead of a UUID?

Reply:
Nope, I already did that! https://github.com/sigstore/cosign/blob/af575bb5a260d9f28bf5a9814c16a7c08923221e/pkg/cosign/tlog.go#L288

there are tests for those too

Reply:
awesome 🎉